tests/e2e/cucumber/environment/index.ts

```diff
@@ -258,3 +261,17 @@ const isOcm = (pickle): boolean => {
   }
   return false
 }
+
+/*
+  store group created from keycloak on store
+ */
+const storeKeycloakGroups = async (user: User, usersEnvironment) => {
+  const groups = await api.graph.getGroups(user)
+
+  dummyKeycloakGroupStore.forEach((dummyGroup) => {
+    const matchingGroup = groups.find((group) => group.displayName === dummyGroup.displayName)
+    if (matchingGroup) {
+      usersEnvironment.storeCreatedGroup({ group: { ...dummyGroup, uuid: matchingGroup.id } })
```

Comment:
is the group id dynamic or static?

Reply:
ocis group id is dynamic and different than keycloak group id